### DISCUSSION TITLE
Trace view - easier for debugging

### DISCUSSION THREAD

**Discussion by ajram23** [langfuse/langfuse#4762] [Ideas]
### Describe the feature or potential improvement

Now that 3.x is out., I wanted to bring up something that needs work. The detail view is pretty painful for debugging especially with the number of spans etc. Use cases where the UI can be helpful with.

1.  If an answer took too long, figuring out which of the LLM calls took too long
2. Same for errors.

Thank you!

**Comment by marcklingen** [comment]
thanks a lot for the feedback @ajram23, improving the core UI is planned for q1 once all v3 cleanups are finished
